src/main/java/org/apache/maven/plugins/checkstyle/CheckstyleViolationCheckMojo.java

```diff
@@ -569,7 +572,9 @@ public void execute()
             XmlPullParser xpp = new MXParser();
             xpp.setInput( reader );
 
-            int violations = countViolations( xpp );
+            final List<Violation> violationsList = getViolations( xpp );
```

Comment:
suggest calling this violations and the count numViolations or some such. "violations" alone does not sound like an integer.

Reply:
Sure, no problem. I did not want to change variable names in this commit, but if you like bigger commits, I am happy to change this.